packages/api/src/controllers/access-control.ts

```diff
@@ -128,17 +125,18 @@ app.post(
 
     const playbackPolicyType = content.playbackPolicy?.type ?? "public";
 
-    if (user.createdAt < HACKER_DISABLE_CUTOFF_DATE) {
-      let limitReached = await freeTierLimitReached(content, user, req);
-      if (limitReached) {
-        throw new ForbiddenError("Free tier user reached viewership limit");
-      }
+    let response = {};
+
+    if (isFreeTierUser(user) && user.createdAt > HACKER_DISABLE_CUTOFF_DATE) {
+      response = {
+        rateLimit: MAX_ALLOWED_VIEWERS_FOR_FREE_TIER,
+      };
     }
 
     switch (playbackPolicyType) {
       case "public":
-        res.status(204);
-        return res.end();
+        res.status(200);
```

Comment:
Make sure catalyst-api is not checking for a 204 explicitly (e.g. just checking for a 2xx). If it is expecting a 204, we need to first deploy a version of catalyst that allows different status codes here.